torchvision/csrc/io/image/cpu/common_jpeg.h

```diff
@@ -1,13 +1,17 @@
 #pragma once
 
-// clang-format off
-#include <cstdio>
-#include <cstddef>
-// clang-format on
-
 #if JPEG_FOUND
+#include <stdio.h>
+
 #include <jpeglib.h>
 #include <setjmp.h>
+#endif
+
+namespace vision {
+namespace image {
+namespace detail {
+
+#if JPEG_FOUND
```

Comment:
Looks like we have two `#if JPEG_FOUND` in this file, but there doesn't seem to be any code not within `JPEG_FOUND`. Should we have instead a single `#if JPEG_FOUND`?

Reply:
I intentionally use two because I wanted to put the headers on top and then actually have the namespace vision::image::detail because I open it from a different file. This way even if it's empty the code will still work. I think I can refactor this to make it work.

Reply:
Fixed!